apps/desktop/src/lib/file/BranchFilesList.svelte

```diff
@@ -126,7 +126,7 @@
 	</div>
 {/if}
 
-{#if displayedFiles.length > 0}
+{#if displayedFiles.length > 0 || (conflictedFiles?.entries.size || 0) > 0}
```

Comment:
`conflictedFiles?.entries.size` if not found is `undefined` and `undefined > 0 === false` apparently, so that would have been fine, but then the editor still complains